packages/block-library/src/pattern/block.json

```diff
@@ -6,13 +6,27 @@
 	"category": "theme",
 	"description": "Show a block pattern.",
 	"supports": {
-		"html": false,
-		"inserter": false
+		"__experimentalLayout": {
+			"allowEditing": false
+		}
 	},
 	"textdomain": "default",
 	"attributes": {
 		"slug": {
 			"type": "string"
+		},
+		"inheritedAlignment": {
```

Comment:
It might be worth moving this change into a separate PR, as I'm sure there will be some other thoughts on it. I think the main thing right now is to get a first implementation of the partial syncing behavior, so this isn't something of high urgency, though it is something that needs to be fixed.

Some random thoughts - I wonder if it could be made part of the `align` block supports. While other blocks might have something like `align: [ 'wide', 'full' ]`, this could possibly be `align: 'auto'`? When set to that it could avoid showing the UI. Just throwing some ideas out there. It might be good to start some discussions with other contributors who may have thought about this problem.

Reply:
As they say 'Insanity is inherited, you get it from your children', so the same could apply here 😄. I have removed it now anyway.  As Dan suggests, probably better as a follow-up PR so as not to slow things down if people have different opinions on it.

Reply:
We might also need to update the PR description to highlight that this has been removed and that the pattern's alignment might not be maintained until that follow-up happens.

Reply:
[PR to fix this here](https://github.com/WordPress/gutenberg/pull/50646)